app/views/api/v1/models/_hook.json.jbuilder

```diff
@@ -1,4 +1,7 @@
 json.id resource.id
-json.app resource.app.params.to_h
-json.enabled resource.enabled?
+json.app_id resource.app_id
+json.status resource.enabled?
```

Comment:
Is this API being using in the frontend anywhere?

Reply:
nope